### PR TITLE
Completed dom-event-delegation exercise - Justin Song

### PR DESCRIPTION
**Pull request by justinsong1608**
Screenshot 2023-02-09 at 9 31 44 AM
Screenshot 2023-02-09 at 9 57 38 AM

### CODE REVIEW

File: dom-event-delegation/main.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+var $taskList = document.querySelector('.task-list');
+
+function target(event) {
+  console.log('event.target:', event.target);
+  console.log('event.target.tagName:', event.target.tagName);
+}
```
Comment (shawnkost) [inline dom-event-delegation/main.js]: I wouldn't create a function just for these console logs. I would move these lines into your `button` function, and just have 1 event listener

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Justin! If you created this function because of what I said yesterday about "separation of concern", I totally get where you were going with this, but in a real life dev environment, you would never submit a pull request with any console logs left in, so I wouldn't recommend creating a function just for these

**Comment by justinsong1608** [comment]
Updated!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**